modules/smart-agent_health-checker/conf/01-health-checker-status.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
+name: status
+id: health_checker_status
```

Comment:
I would remove this to keep the variables names shorter with `status` only.

Reply:
done